stdlib/SuiteSparse/test/umfpack.jl

```diff
@@ -26,6 +26,7 @@ using LinearAlgebra: Adjoint, Transpose, SingularException
             @test (Diagonal(Rs) * A)[p,q] ≈ L * U
 
             det(lua) ≈ det(Array(A))
+            logabsdet(lua) ≈ logabsdet(Array(A))
```

Comment:
Doesn't there need to be a `@test` in front of these?   Also, you need to check the sign separately, no?